requirements.txt

```diff
@@ -25,3 +24,4 @@ optree>=0.13.0
 psutil
 sympy>=1.13.3
 typing-extensions>=4.13.2
+wheel
```

Comment:
Why is it needed here for generic requirements?

Reply:
It is a development dependency:

https://github.com/pytorch/pytorch/blob/9508d73307b4bc1fe453677526a096e5e10a7575/.ci/docker/common/install_cpython.sh#L69-L71

https://github.com/pytorch/pytorch/blob/9508d73307b4bc1fe453677526a096e5e10a7575/tools/nightly.py#L584-L608